_layouts/default.html

```diff
@@ -61,9 +61,6 @@
         <paper-item label="Developer API (javadoc)" icon="open-in-browser">
           <a href="api/latest/" target="_blank"></a>
         </paper-item>
-        <paper-item label="Releases &amp; Versioning">
```

Comment:
I'm not sure why it's only happening now... but I misread you when you said "only committed externally".  That one surprises me.  you could be right, and it's just a matter of the manual fiddling to get the initial export didn't blow it up, but now that we're syncing, its being absent in our internal copy is being faithfully replayed. :(